pytrip/tripexecuter/plan.py

```diff
@@ -531,6 +531,8 @@ def _make_exec_opt(self):
         # TODO: sanity check numbers
         opt += " geps({:.2e})".format(self.geps)
         opt += " eps({:.2e})".format(self.eps)
+        opt += "iter({:d})".format(self.iterations)
```

Comment:
You will need a leading space before iter(). I doubt TRiP parser would accept this...

Reply:
Ofc! My bad!

Reply:
New code has been committed